### PR TITLE
Add slider

### LINKED ISSUE #3650: Expenditures: As a user I can use a slider to adjust recipient share on "Unequal" split
[Provided by Adrian via Contractors project management platform]

## Description
When toggle switch is on "Unequal", user has to be able to display and use progress slider. A slider bar will allow to set a percentage usage of max funds.

## Acceptance Criteria

- an option to display and use a slider
- an option to display percent

## Design

- [Figma Link - Allocate funds slider](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3807%3A285480)

da3a5f92a471066199c32743d977b921

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description
This PR adds a progress slider that is visible when the switch is in the "Unequal" position. A slider bar allows to set a percentage usage of max funds.

Screenshot 2022-07-25 at 07 49 16

**Changes** 🏗

* Changed Slider component by adding possibility to override styling. It has been done by adding four optional props in Slider component:  
```
  handleStyle?: React.CSSProperties;
  dotStyle?: React.CSSProperties;
  railStyle?: React.CSSProperties;
  trackStyle?: React.CSSProperties;
```
These objects override the default styles:
```
 trackStyle={{
          backgroundColor: colors.backgroundColor,
          height: sizes.height,
          ...trackStyle,
        }}
```

## TODO

- add the rest of "Unequal" form in the next PR

Resolves #3650

### CODE REVIEW

**Review by willm30 — CHANGES_REQUESTED** [review]
@agnieszkajarosikloj Works nicely 🙂Noticed a few small things. 

1. Percentage label is a little too close to slider when 100%

too-close

1. There are some lines visible in the sliders when the sum is below 100%

lines-vis

1. I think the form should be disabled if the sum of the sliders is under 100%?

File: src/modules/dashboard/components/ExpenditurePage/Split/Split.tsx [hunk 1]
```diff
@@ -4,8 +4,10 @@ import { defineMessages, FormattedMessage } from 'react-intl';
 import classNames from 'classnames';
 
 import { FormSection, Toggle } from '~core/Fields';
+import { Colony } from '~data/index';
 
 import styles from './Split.css';
+import { SplitUnequal } from '.';
```
Comment (danbr) [inline src/modules/dashboard/components/ExpenditurePage/Split/Split.tsx:10]: Usually style imports last

**Review by danbr — APPROVED** [review]
Looks good, working well for me. well done.

**Review by willm30 — APPROVED** [review]
Nice one 👍

**Pull request merged**